packages/ember-views/lib/components/component.js

```diff
@@ -158,6 +159,25 @@ var Component = View.extend(TargetActionSupport, {
 
       this.layout = this.defaultLayout;
     }
+
+    // If in a tagless component, assert that no event handlers and custom eventManager are defined
+    assert(
+      'You can not define a function that handles DOM events in a tagless ' +
+      'component since it doesn\'t have any DOM element.',
```

Comment:
I'd like to include the name of the component (basically just `${this}`) which should indicate the module name.